editor/src/uuiui/styles/theme.ts

```diff
@@ -190,8 +158,154 @@ const light = {
   flasherHookColor: base.neonpink,
 }
 
+/** DARK **/
+const darkBase = {
+  darkPrimary: base.blue.shade(150),
+  primary: base.blue,
+  component: base.purple,
+  componentChild: base.orange,
+  css: base.neongreen,
+  white: base.white,
+  transparent: base.transparent,
+  brandPurple: base.purple,
+  brandNeonYellow: base.neonyellow,
+  brandNeonPink: base.neonpink,
+  spectrumTop: base.white,
+  spectrumBottom: base.black,
+  bg0: createUtopiColor('#000000', 'brightest', 'white'),
+  bg1: createUtopiColor('#181C20', '', 'white'),
+  bg2: createUtopiColor('#373C4A', 'eg inputs', 'white'),
+  bg3: createUtopiColor('#55575F', 'visible buttons', 'white'),
+  bg4: createUtopiColor('#4C4D5B', 'hover states for buttons', 'white'),
+  bg5: createUtopiColor('#848998', 'selected elements', 'grey'),
+  fg0: createUtopiColor('#ffffff', 'emphasized foreground', 'black'),
+  fg1: createUtopiColor('#D9DCE3', 'default foreground', 'black'),
+  // fg2: createUtopiColor('red', 'black', 'black'),
+  fg3: createUtopiColor('pink', 'darkgray', 'darkgray'),
+  // fg4: createUtopiColor('orange', 'darkgray', 'darkgray'),
+  fg5: createUtopiColor('#8B91A0', 'leva', 'grey'),
+  fg6: createUtopiColor('#6F778B', 'grey', 'grey'),
+  fg7: createUtopiColor('#525B72', 'grey', 'grey'),
+  fg8: createUtopiColor('#2F374A', 'lightgrey', 'lightgrey'),
+  fg9: createUtopiColor('#151A27', 'lightgrey', 'lightgrey'),
+  border1: createUtopiColor('#181C20', '', ''),
+  border2: createUtopiColor('#181C20', '', ''),
+  border3: createUtopiColor('#181C20', '', ''),
+}
+
+const darkPrimitives = {
```

Comment:
Follow up for @maltenuhn - I guess you'll want to update the values used here? I'll create a ticket

Reply:
#1436